addons/func_godot/src/netradiant_custom/netradiant_custom_gamepack_config.gd

```diff
@@ -49,6 +49,19 @@ extends Resource
 ## Skip texture path that gets applied to caulk and nodrawnonsolid shaders.
 @export var skip_texture: String = "textures/special/skip"
 
+## Variables to include in the exported gamepack's [code]default_build_menu.xml[/code].[br][br]
+## Each [String] key defines a variable name, and its corresponding [String] value as the literal command-line string to execute in place of this variable identifier[br][br]
+## Entries may be referred to by key in [member default_build_commands] values.
+@export var default_build_variables: Dictionary
+
+## Commands to include in the exported gamepack's [code]default_build_menu.xml[/code].[br][br]
+## Keys, specified as a [String], define the command name as you want it to appear in Radiant.[br][br]
+## Values represent steps taken within the build command.[br][br]They may be either a [String] or an
+## [Array][[String]] and will be used as the full command-line text issued by each step [i]within[/i]
+## its associated build command key. [br][br]They may reference entries in [member default_build_variables]
+## by using brackets: [code][variable key name][/code]
+@export var default_build_commands: Dictionary
```

Comment:
As above, should be `default_build_menu_commands` as it gets applied to the default_build_menu.xml.